domovoi.py

```diff
@@ -60,7 +60,7 @@ def run(self, processes_file):
         # Watch over the processes and respond accordingly
         while True:
             for solar_car_process in solar_car_processes:
-                if solar_car_process.check_status() is not None:
+                if solar_car_process.check_status() != None:
```

Comment:
As discussed before the most pythonic way of checking for None is, `if solar_car_process.check_status():`